x-pack/plugins/cases/public/components/configure_cases/index.tsx

```diff
@@ -526,6 +613,26 @@ export const ConfigureCases: React.FC = React.memo(() => {
       </CommonFlyout>
     ) : null;
 
+  const AddOrEditObservableTypeFlyout =
```

Comment:
We should include some limit to the number of Observables Types that can be created. Right now there seems to be no limit.